### ISSUE TITLE
Blockchain API Receive callbacks are arriving too late

### ISSUE THREAD

**Issue by arif7576** [blockchain/receive-payments-demos#78]
Payment  my callback url called delay. for example, i transfer the amount on the 12:00 am, and my callback url called 4-5 hours delay.

**Comment by ConKreter** [comment]
yes I have exactly the same issue. Its super delayed and they come many times even after *ok*

**Comment by egorix** [comment]
Same here guys. More than 24 hours delay! Any news on that?

**Comment by iamandyagbaraku** [comment]
> Same here guys. More than 24 hours delay! Any news on that?

Hey, @egorix @ConKreter @arif7576, are you still having the same issue? If no, how did you fix it?

**Comment by arif7576** [comment]
Not perfect, sometime work good , sometime not working

**Comment by ninob43** [comment]
I fixed it  by making txhash unique
$sql="select * from payment where txhash='$txhash'";
$result = mysqli_query($link, $sql);
   if ($result->num_rows > 0){
              
                       while ($row = $result->fetch_assoc()){
                           $stat=1;
                          
                          
                       }}

if(stat==1){}else{
//update your payments

}
